content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -26,7 +26,7 @@ It also offers full support for React Hooks, including inspecting nested objects
 * `0`-`0.61`: Not supported
 * `0.62`: Will be supported (when 0.62 is released)
 
-## How do I get the new DevTools?
+## How do I get the new DevTools? {#how-do-i-get-the-new-devtools}
```

Comment:
same as above

`## How do I get the new DevTools? {#how-do-i-get-the-new-devtools}` => `## How do I get the new DevTools?`